gleam.toml

```diff
@@ -9,6 +9,7 @@ repository = { type = "github", user = "ninanomenon", repo = "shork" }
 gleam_erlang = ">= 0.34.0 and < 1.0.0"
 gleam_stdlib = ">= 0.54.0 and < 1.0.0"
 mysql = ">= 1.8.0 and < 2.0.0"
+gleam_otp = ">= 0.16.1 and < 1.0.0"
```

Comment:
This MR is open for so long, that in the mean time 1.0 of gleam_otp was published: https://github.com/gleam-lang/otp/releases/tag/v1.0.0